libs/wingsdk/src/target-sim/topic.inflight.ts

```diff
@@ -81,17 +80,20 @@ export class Topic
     }
   }
 
-  public async publish(message: string): Promise<void> {
-    this.context.addTrace({
-      data: {
-        message: `Publish (message=${message}).`,
+  public publish(...messages: string[]): Promise<void> {
```

Comment:
please add a few tests at `libs/wingsdk/test/target-sim/topic.test.ts` and `libs/wingsdk/test/shared-aws/topic.inflight.test.ts`